docs/source/dev/environment.md

```diff
@@ -167,6 +167,14 @@ pnpm --prefix ui start
 
 The above will automatically open a browser with the URL: <http://localhost:5173>
 
+> **Note**: In case that you decided to install MXCuBE-Web without using the `conda-environment.yml` file, the stream in the `Data Collection` tab of the UI might be missing. To resolve this:
+>
+> 1. Ensure that `ffmpeg` is properly installed on your side, to install run `sudo apt install ffmpeg`
+>    Then restart the server.
+> 1. Alternatively, you can manually install the dependencies of [MXCuBE's video-streamer](https://github.com/mxcube/video-streamer/) and restart the server afterwards.
+>
+> If the issue persists, please make sure to use the [issues](https://github.com/mxcube/mxcubeweb/issues) page and reach out for help.
```

Comment:
The documentation is completely written around this `conda-environment.yml` file.
I really do not wish for us to have to maintain documentation for more than one scenario.
If it were just me I would not put this note in.

Better would be to write a section about what the dependencies of MXCuBE-Web are,
besides the dependencies that are already obvious because they are written in `pyproject.toml` and `poetry.lock`.
And this section could be just as short as:

> Please, look into the `conda-environment.yml` for the list of additional dependencies.

Reply:
I think it was more meant as a troubleshooting, since a few people seemed to have the problem and the application doesn't really tell you about the missing dependency. 

Sure, a dependency list can be a very good addition to the documentation. 
Actually, I was wondering, if we should add some sort of FAQ to the documentation (maybe this would go over the topic of this PR)

Reply:
I guess it was an issue because `ffmpeg` was not listed anywhere as a dependency. If `ffmpeg` is now listed as a dependency in `conda-environment.yaml`, then it will be installed and people will not have the issue of `ffmpeg` missing. Am I misunderstanding something? :D

I see that `ffmpeg` is not listed here either:
https://github.com/mxcube/video-streamer/blob/539e65b34a8d6493945cc886957d05242cf1858b/conda-environment.yml
Is that normal?

Anyway, we can keep the note, I am not convinced it is necessary, but obviously it is not a big deal.